.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.4.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-case-conflict
+  - id: check-json
+  - id: check-merge-conflict
+  - id: check-symlinks
+  - id: check-toml
+  - id: end-of-file-fixer
```

Comment:
Wow, first time I hear of these.

Reply:
I heard about them a while back and we're using a few in wagtail-grapple. 
Here's the full list: https://pre-commit.com/hooks.html